packages/integration-karma/test/synthetic-shadow/inner-outer-text/inner-outer-text.spec.js

```diff
@@ -0,0 +1,150 @@
+import { createElement } from 'lwc';
+import Container from 'x/container';
+
+if (!process.env.NATIVE_SHADOW) {
```

Comment:
Are all tests failing in native shadow? This means the patch is diverging from native behavior widely.